tools/lambda-promtail/lambda-promtail/aws_secrets.go

```diff
@@ -0,0 +1,81 @@
+package main
+// Use this code snippet in your app.
+// If you need more information about configurations or implementing the sample code, visit the AWS docs:   
+// https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/setting-up.html
+
+import (
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"encoding/json"
+	"fmt"
+)
+
+
+func getSecret(secretName,usernameKey, passwordKey, region string) (username, password string) {
```

Comment:
Can you rename this to `getAWSSecret()`? It's clearer that it's only for AWS that way.